### PR TITLE
Switch to salesforce-sdk 1.1.1

### PR DESCRIPTION
**Pull request by dhagberg-sf**
- Bump sdk version, pin with `^` to allow using latest compatible version rather than exact
- bump buildpack.toml

### CODE REVIEW

File: middleware/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sf-fx-middleware",
-  "version": "0.0.1",
+  "version": "0.0.2",
```
Comment (jqian2017) [inline middleware/package.json]: even though we're not using this version right now.. but I wonder if we should keep consistent with the biuldpack.toml version... or remove it?

File: middleware/package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
   },
   "author": "TODO",
   "dependencies": {
-    "@salesforce/salesforce-sdk": "1.1.0",
+    "@salesforce/salesforce-sdk": "^1.1.1",
```
Comment (jqian2017) [inline middleware/package.json]: thank you for ^

File: middleware/package.json (hunk 1 above)
Comment (dhagberg-sf) [reply middleware/package.json]: Yeah, I wasn't sure if this was used - makes sense to keep these in sync.

Comment (dhagberg-sf) [reply middleware/package.json]: Fixed.

**Pull request merged**